src/Utility/Xml.php

```diff
@@ -108,7 +109,17 @@ public static function build($input, array $options = [])
             'readFile' => true
         ];
         $options += $defaults;
-
+        
+        if (!filter_var($input, FILTER_VALIDATE_URL) === false) {
+            $http = new Client();
+            $response = $http->get($input);
+            $input = $response->body();
+        }
```

Comment:
This is a huge security issue. That's why we removed this feature. Imagine a scenario where you are accepting an XML string from a user. They can now have your server do a request to their misbehaving server and easily perform a denial of service attack.